src/lib/message.ts

```diff
@@ -189,6 +189,20 @@ class GleeMessage extends EventEmitter {
   send() {
     this.emit('send', this)
   }
+
+  /**
+   * Indicates successfully processed the message
+   */
+  notifySuccessfulProcessing() {
+    this.emit('successfullyProcessed')
```

Comment:
Let's follow the structure we're using everywhere else in the code base:

```suggestion
    this.emit('processing:successful')
```